index.html

```diff
@@ -6,27 +6,61 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="stylemobile.css">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Poppins" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body> 
  <section class="first-page">  
-  <header >        
-        <nav class="navbar" >
+  <header >  
+    <nav class="navbar" >
+      <a href=""><img class="mylogo" src="images\My Logo.png"  alt="My Logo"></a>
+      <a href=""><img src="images\Union.png" alt="Menu"></a>             
+   </nav>   
+        <nav class="navbard" >
             <a href=""><img class="mylogo" src="images\My Logo.png"  alt="My Logo"></a>
-            <img class="menu" src="images\Union.png" alt="Menu"> 
+            <ul>
+              <li>Portfolio</li>
+              <li>About</li>
+              <li>Contact</li>
+          </ul>      
         </nav>       
   </header>
   <main>
-    <section class="headline">
-           
-            <div id="container-two">
+    <section class="headline-mobile">
+      <div class="container-one">
+        <h1 class="introduction">
+          I’m Abiy.<br>
+          Glad to see <br> you!
+        </h1>
+        <p class="profession">
+          I’m a software developer! I can help you<br> build a product , feature or website Look <br>through some of my work and<br> experience! If you like what you see and<br> have a project you need coded, don’t <br>hesitate to contact me.
+        </p>
+        <h3 class="motto" >
+          LET’S CONNECT
+        </h3>
+        <div class="social-media">
+          <ul>
+              <li><a ><img class="twitter icons"src="images\Vector.svg" alt="twitter"></a></li>
+              <li><a><img class="linkedin icons" src="images\Linkedin icon@2x (1).png" alt="linkedin"></a></li>
+              <li><a><img class="vector-1 icons" src="images\Vector (1).svg" alt=""></a></li>
+              <li><a><img class="vector-2 icons" src="images\Vector (2).svg" alt="github"><a></a></li>
+              <li><a><img class="vector-3 icons" src="images\Vector (3).svg" alt=""><a></li>
+          </ul>              
+        </div>
+        
+      </div>
+    </section>
+        
+    <section class="headline" id="headline-desktop">
```

Comment:
- [ ] Please I highly recommend sticking to the [Figma Design Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=Hq64bUbN7WFDPuHZ-0). 😄 

_**REQUIRED FIGMA DESIGN**_

![Screenshot (195)](https://user-images.githubusercontent.com/78104544/230508854-8ba9d4ea-d177-487f-a326-a8780184d5cf.png)

_**YOUR DESIGN**_

- [ ] Please I recommend adjusting the margin or padding top of the `navbar` class as I have numbered it 1 to fit the Figma Design. 😃 
- [ ] Please I recommend adjusting the designs of the `container-one` class with `heights` and `margins` as I have numbered it 2 to fit the Figma Design. 😄 

![Screenshot (196)](https://user-images.githubusercontent.com/78104544/230509223-8542f4a6-53a2-4cd1-9c40-9c3e8670ade2.jpg)